.prettierrc

```diff
@@ -0,0 +1 @@
+proseWrap: always
```

Comment:
was this supposed to be committed?

Reply:
Yeah, I guess I could split it out into another PR but it sure makes life easier.